charts/komodor-agent/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+name: komodor-agent
+description: Watches and sends kubernetes resource-related events
+icon: https://raw.githubusercontent.com/komodorio/helm-charts/master/k8s-watcher.svg
+version: 1.17.0
+appVersion: 0.2.59
```

Comment:
Let's make sure we are on the latest agent version when this is merged -- Also, I saw in the pipeline.yml we update the agent version here as well so that's good.